notebooks/00_quick_start/wide_deep_model_movielens.ipynb

```diff
@@ -0,0 +1,1388 @@
+{
```

Comment:
<p>Can you move these parameters to the top? like in the other notebooks? We will use this to control the parameters in the unit tests. I have to write some documentation on how to add parameters with papermill, so the notebooks can be executed in the tests</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Make sense to `NUM_EPOCHS` where we can pass very small number for unit-testing, will do that. But wonder about other hyperparameters. Wouldn't it be better flow to keep them under the modeling section?

Reply:
Added to the top cell, so that aml notebook also can utilize this wide-deep notebook for its model training/testing via papermill's parameterizing.